src/app/asset-grid/thumbnail/thumbnail.component.ts

```diff
@@ -95,14 +98,16 @@ export class ThumbnailComponent implements OnInit, OnChanges {
     // Prevent the parent anchor tag from firing
     event.preventDefault()
     event.stopPropagation()
-
-    if (urlParams[0] === '/associated') {
-      this.angulartics.eventTrack.next({ properties: { event: 'view associated images', label: this.thumbnail.id } })
-    }
-    if (urlParams[0] === '/cluster') {
-      this.angulartics.eventTrack.next({ properties: { event: 'view cluster', label: this.thumbnail.id } })
+    // Don't navigate to other page in compare mode
+    if (!this.compareMode) {
```

Comment:
Have we talked with Nick about this behavior? Preventing this link from being clickable seems a little weird especially since you can tab to them. Wondering if we still wanted to show them in the first place?

Reply:
I haven't, I'm going to ask him when he's available.